packages/ckeditor5-engine/tests/conversion/upcasthelpers.js

```diff
@@ -969,6 +969,31 @@ describe( 'upcast-converters', () => {
 			expect( conversionResult.getChild( 0 ).data ).to.equal( 'foobar' );
 		} );
 
+		it( 'should wrap text with paragraph if it would allow conversion', () => {
```

Comment:
Maybe 'should auto-paragraph text if it is not allowed at insert postion but is inserted where paragraph is allowed'?